docs/README.md

```diff
@@ -76,16 +77,19 @@ For example:
     "markers": [
       "highlight"
     ],
-    "blockClass": "summary"
+    "blockClass": "summary",
+    "message": "{sellingPriceValue}"
   }
 },
 ```
 
-### Step 3 - Editing the block's message  on Site Editor
+### Step 3 - Editing the block's message on Site Editor or in the message prop
 
 Every Product Price's block uses the [ICU Message Format](https://format-message.github.io/icu-message-format-for-translators/), making it possible to fully edit the text messages and the variables displayed by each block.
 
-Access your VTEX account's admin, open the Site Editor and look for the Product Price's blocks. 
+To edit the messages with the block message prop follow the exemple in the above section using the avalible variables bellow. 
+
+To edit the messages in the Site Editor access your VTEX account's admin, open the Site Editor and look for the Product Price's blocks. 
 
 ![Product-Price-Site-Editor-gif](https://user-images.githubusercontent.com/52087100/78073694-bdbffd80-7377-11ea-9262-40854dccdd53.gif)
```

Comment:
```suggestion
```

Reply:
Don't know why, but I'm not being able to delete the next paragraph as well. Could you do this for me? 😃

Reply:
Sure. Done!